src/main/java/com/epam/ta/reportportal/core/item/impl/FinishTestItemHandlerImpl.java

```diff
@@ -139,13 +139,15 @@ private TestItemResults processItemResults(Long projectId, TestItem testItem, Fi
 					IssueType issueType = issueTypeHandler.defineIssueType(testItem.getItemId(), projectId, locator);
 					issueEntity = IssueConverter.TO_ISSUE.apply(providedIssue);
 					issueEntity.setIssueType(issueType);
+					issueEntity.setIssueId(testItem.getItemId());
+					testItemResults.setIssue(issueEntity);
 				}
 			} else {
 				IssueType toInvestigate = issueTypeHandler.defineIssueType(testItem.getItemId(), projectId, TO_INVESTIGATE.getLocator());
 				issueEntity.setIssueType(toInvestigate);
+				issueEntity.setIssueId(testItem.getItemId());
+				testItemResults.setIssue(issueEntity);
 			}
-			issueEntity.setIssueId(testItem.getItemId());
```

Comment:
why did you moved it into if and else structure?
it should always be called

Reply:
if we finish item with status skipped -> to item results sets empty issue only with id -> hibernate trying to save it and throws constraint violation exception